application/pom.xml

```diff
@@ -481,6 +481,18 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <pluginRepositories>
```

Comment:
Is this artifactory necessary?  I believe we have pushed all of the needed artifacts to maven central, if I recall correctly.

Reply:
We pushed the WebAPI to Mavencentral.
However, the repository is needed for JPro itself.
We are still talking internally about whether we should move the JPro plugins.
But that shouldn't matter too much.
The repositories are hidden under a Profile.
When JPro is used, our repository is used anyway.
It shouldn't influence the desktop build.

I Hope that's fine on your side.

Reply:
@FlorianKirmaier 
I believe this is isolated in a profile and likely fine. Our main concern is making sure it does not prevent customers from building Komet publicly.

We will merge this PR for now 👍🏼 . And later revisit with a follow on ticket if or when your team decides to move libraries.